src/common/layout/__tests__/__snapshots__/PageLayout.test.tsx.snap

```diff
@@ -1,32 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`matches snapshot 1`] = `
-<HelmetProvider
-  context={Object {}}
->
-  <Helmet
-    defer={true}
-    encodeSpecialCharacters={true}
-  >
-    <html
-      lang="fi"
-    />
-    <title>
-      Nuorisopalvelut
-    </title>
-    <meta
-      content="Jäsenkortilla pääset mukaan nuorisotalojen toimintaan. Jässärin omistajana voit viettää iltaa discossa tai yökahvilassa, hengailla nuorisotalolla, harjoitella bändikämpillä, käydä dj-kurssin, harrasta tanssia, teatteria tai kädentaitoja, hakea projektirahaa, osallistua kesäleireille tai vaikka tuottaa oman tapahtuman."
-      name="description"
-    />
-    <meta
-      content="Nuorisopalvelut"
-      property="og:title"
-    />
-    <meta
-      content="Jäsenkortilla pääset mukaan nuorisotalojen toimintaan. Jässärin omistajana voit viettää iltaa discossa tai yökahvilassa, hengailla nuorisotalolla, harjoitella bändikämpillä, käydä dj-kurssin, harrasta tanssia, teatteria tai kädentaitoja, hakea projektirahaa, osallistua kesäleireille tai vaikka tuottaa oman tapahtuman."
-      property="og:description"
-    />
-  </Helmet>
+<PageWrapper>
```

Comment:
Is it so that we no longer have a snapshot that contains the Helmet page head? An observation, nothing else.

Reply:
I added a quick snapshot test for PageWrapper.